todoman/cli.py

```diff
@@ -336,6 +337,9 @@ def list(
 
     sort = sort.split(',') if sort else None
 
+    if priority not in ['medium', 'low', None, 'high']:
+        raise "Invalid priority"
```

Comment:
You can't raise a string. Also I'd prefer if validation of this parameter is done in an extra callback (see e.g. `_validate_list_param`.

Also I'd prefer if we translate those strings to numbers in this file, not in `model.py`.